pom.xml

```
@@ -32,7 +32,7 @@
     <dependency>
       <groupId>com.github.vandeseer</groupId>
       <artifactId>easytable</artifactId>
-      <version>0.6.6</version>
+      <version>1.0.2</version>
```

This version is vulnerable click here to commit fix.
```suggestion
      <version>1.0.6</version>
```